crates/core/src/abi.rs

```diff
@@ -894,84 +904,62 @@ impl<'a, B: Bindgen> Generator<'a, B> {
                     self_.stack.push(ptr);
                 };
 
-                if async_ {
+                if !sig.indirect_params {
+                    // If the parameters for this function aren't indirect
+                    // (there aren't too many) then we simply do a normal lower
+                    // operation for them all.
+                    for (nth, (_, ty)) in func.params.iter().enumerate() {
+                        self.emit(&Instruction::GetArg { nth });
+                        self.lower(ty);
+                    }
+                } else {
+                    // ... otherwise if parameters are indirect space is
+                    // allocated from them and each argument is lowered
```

Comment:
"from them" or "for them"? Maybe it's as intended.

Reply:
Yeah, I think this should be "for them"

```suggestion
                    // allocated for them and each argument is lowered
```